packages/sample-controllers/src/sample-gas-prices-service/sample-gas-prices-service.test.ts

```diff
@@ -26,3 +294,65 @@ describe('SampleGasPricesService', () => {
     });
   });
 });
+
+/**
+ * The type of the messenger where all actions and events will be registered.
+ */
+type UnrestrictedMessenger = Messenger<
```

Comment:
I need to follow the same changes in https://github.com/MetaMask/core/pull/6335 with regard to setting up the messenger in tests.